app/javascript/layouts/tailwind.css

```diff
@@ -144,17 +141,17 @@ button:focus {
   @apply bg-primary-500 text-white shadow;
   background-image: linear-gradient(
     135deg,
-    rgba(96, 37, 192, 1) 0%,
-    rgba(77, 30, 154, 1) 100%
+    theme("colors.primary.500") 0%,
+    theme("colors.primary.600") 100%,
```

Comment:
@mpraglowski TailwindCSS supports `bg-gradient` utility. We can use
`@apply bg-gradient-to-br from-primary-500 to-primary-600` instead of writing custom CSS.